docs/products/kafka/kafka-connect/howto/debezium-source-connector-mysql.rst

```diff
@@ -0,0 +1,146 @@
+* ``MYSQL_HOST``, ``MYSQL_PORT``, ``MYSQL_DATABASE_NAME``, ``SSL_MODE``, ``MYSQL_USER``, ``MYSQL_PASSWORD``, ``MYSQL_TABLES``: source database parameters collected in the :ref:`prerequisite <connect_debezium_mysql_source_prereq>` phase. 
+* ``database.server.name``: the logical name of the database, dictates the prefix that will be used for Apache Kafka topic names. The resulting topic name will be the concatenation of the ``database.server.name`` and the table name.
+* ``tasks.max``: maximum number of tasks to execute in parallel. By default this is 1, the connector can use at most 1 task for each source table defined. Replace ``NR_TASKS`` with the amount of parallel task based on the number of tables.
+* ``database.history.kafka.topic``: the name of the Apache Kafka topic that will contain the history of schema changes.
+* ``database.history.kafka.bootstrap.servers``: points to the Aiven for Apache Kafka service where the connector is running and is needed to store :ref:`schema definition changes <connect_debezium_mysql_schema_versioning>`
+* ``database.history.producer`` and ``database.history.consumer``: points to truststores and keystores pre-created on the Aiven for Apache Kafka node to handle SSL authentication
+
+  .. Warning::
+
+    The values defined for each ``database.history.producer`` and ``database.history.consumer`` parameters are already set to work with the predefined truststore and keystore created in the Aiven for Apache Kafka nodes. Therefore, they **should not be changed**.
+
+* ``key.converter`` and ``value.converter``:  defines the messages data format in the Apache Kafka topic. The ``io.confluent.connect.avro.AvroConverter`` converter pushes messages in Avro format. To store the messages schema we use Aiven's `Karapace schema registry <https://github.com/aiven/karapace>`_ as specified by the ``schema.registry.url`` parameter and related credentials.
+
+  .. Note::
+
+    The ``key.converter`` and ``value.converter`` sections are only needed when pushing data in Avro format. If omitted the messages will be defined in JSON format.
+
+    The ``USER_INFO`` is not a placeholder, no substitution is needed for that parameter.
+
+
+Create a Kafka Connect connector with the Aiven Console
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To create the connector, access the `Aiven Console <https://console.aiven.io/>`_ and select the Aiven for Apache Kafka® or Aiven for Apache Kafka Connect® service where the connector needs to be defined, then:
+
+1. Click on the **Connectors** tab
+2. Clink on **Create New Connector**, the button is enabled only for services :doc:`with Kafka Connect enabled <enable-connect>`.
+3. Select the **Debezium - MySQL**
+4. Under the *Common* tab, locate the **Connector configuration** text box and click on **Edit**
+5. Paste the connector configuration (stored in the ``debezium_source_mysql.json`` file) in the form
+6. Click on **Apply**
+
+   .. note::
```

Comment:
There is different indentation between this note and tip below, even though for me they should be on the same level. Could you double check if this is intentional?